frontend/hub/remotes/RemoteDetails.tsx

```diff
@@ -103,6 +104,13 @@ export function RemoteDetails() {
             )}
           </PageDetail>
         </PageDetails>
+        <PageDetails numberOfColumns="single">
+          <PageDetailCodeEditor
+            label={t('YAML requirements')}
+            value={remote?.requirements_file ?? '---'}
+            showCopyToClipboard={true}
```

Comment:
should this be disabled when requirements_file is falsy? (copy to clipboard or the whole code editor?)

Reply:
@himdel +1